configure.ac

```diff
@@ -366,6 +368,11 @@ else
 	CPPUTEST_CPPFLAGS="${CPPUTEST_CPPFLAGS} -include ${srcdir}/include/CppUTest/MemoryLeakDetectorMallocMacros.h"
 fi
 
+# Disablong long long support
+if test "x${use_longlong}" = xno; then
+  AC_DEFINE([DISABLE_LONG_LONG], 1, [enable long long])
+fi
```

Comment:
Agree with @uecasm about the name. Tiny typo in the comment, too - `Disablong` -> `Disabling`